packages/@tinacms/datalayer/src/gitProviders.ts

```diff
@@ -0,0 +1,84 @@
+import { Octokit } from '@octokit/rest'
```

Comment:
might be good to use the same pattern as bridge, with a `provider` directory, and then each implementation in a separate module: https://github.com/tinacms/tinacms/tree/main/packages/%40tinacms/graphql/src/database/bridge